applications/areg-portal/frontend/src/components/NavBar/Navbar.tsx

```diff
@@ -13,16 +13,23 @@ import Searchbar from "./Searchbar";
 import NavLinks from "./NavLinks";
 import HelpMenu from "./HelpMenu";
 import { useUser, User } from "../../services/UserService"
+import UserAccountMenu from "./UserAccountMenu";
 
 const Navbar = () => {
   const user: User = useUser();
-  console.log(user);
+  const [isLoggedIn, setIsLoggedIn] = React.useState<boolean>(false)
   const login = () => {
-    
+
     window.location.href = "/login";
+    setIsLoggedIn(true);
```

Comment:
remove: after you change page does not have effect to update the state or anything else